### DISCUSSION TITLE
User email and info on Google Log in

### DISCUSSION THREAD

**Discussion by irg1008** [go-pkgz/auth#180] [Q&A]
Hi, I see in the ClaimsUpd function inside the auth Options, that the claims when signing with google are not being populated:

This is the response of the claims:
```json
{
  "name": "Iván RG",
  "id": "google_96*************e9dc45708106",
  "picture": "http://localhost:8000/avatar/"
}
```

I set the scopes of google sign in to provide user information but it's not being injected into the claims

Thank you

**Comment by umputun** [comment]
The ClaimsUpd has little to do with the scopes and additional info you'd like. This is [how it is usually used](https://github.com/umputun/remark42/blob/master/backend/app/cmd/server.go#L1151), and as you can see, it is a way to set custom attributed to the token

For extra scope support, you should check https://github.com/go-pkgz/auth/pull/163 (just merged), allowing you to do what you need. Please note - this support in master only.

**Comment by irg1008** [comment]
Is there any plan to release a new version with these changes soon? Just to mark this as answered

Thank you
